build-system/single-pass.js

```diff
@@ -391,6 +400,16 @@ function setupBundles(graph) {
   });
 }
 
+/**
+ * Returns true if the file is known to be a common JS module.
+ * @param {string} file
+ */
+function isCommonJsModule(file) {
+  return commonJsModules.some(function(module) {
```

Comment:
If we decide to do this long term, an `acorn` parse/walk can give us the answer for the module pretty easily.

https://astexplorer.net/#/gist/757ee81574c580090685c1c307013048/564cf4d26814ea082d86d11e2e7a0dea2d3ee64f